source/docs/software/commandbased/commands-v3/migration-from-v2.rst

```diff
@@ -0,0 +1,85 @@
+# Migrating from Commands v2 to v3
+
+.. todo::
+   This article guides teams through migrating an existing Commands v2 codebase to v3. Migration can be done incrementally, and this article should help teams plan their migration strategy, understand the key differences, and avoid common pitfalls. This is a critical resource for established teams.
+
+## Migration Strategy
+
+.. todo::
+   Provide high-level guidance on planning a v2-to-v3 migration. Cover:
+   - Should teams migrate? (pros/cons, when to migrate vs. stay on v2)
+   - Migration approaches: full rewrite vs. incremental migration
+   - Incremental migration: v2 and v3 can coexist in the same project
```

Comment:
Incremental migration seems like a massive footgun. This isn't something we should recommend to teams